### ISSUE TITLE
Remove doc parity monitoring tests

### ISSUE THREAD

**Issue by matschaffer** [elastic/elastic-stack-testing#1153]
Starting with 8.0 we've accepted that monitoring docs will differ between versions. We have aliases in place to make sure queries work across versions, but we do not expect perfect field-for-field parity between collection methods.

This likely means just removing the https://github.com/elastic/elastic-stack-testing/tree/main/playbooks/monitoring tree and all associated jenkins jobs.

**Comment by liza-mae** [comment]
I have a PR up in infra repo to remove the jenkins jobs for monitoring, once that is merged, I will put another PR to remove the playbooks from this repo.
